f2/utils/azurescript

```diff
@@ -1,10 +1,11 @@
  export PROJECT_NAME=rcmpcybercrime
  export RG_NAME=MpSCCDSCybercrimeRG
  export ACR_NAME=MpSCCDSCybercrimeacr
- export IMAGE_NAME=cybercrimeimage
+ export IMAGE_NAME=f2
```

Comment:
at some point we should rename our "f2" to "frontend". Have to remember to change image names at that time.

Reply:
Yes, let me know when f2 is renamed to "frontend" so that we can reconfigure a) snyk b) the container settings for the app service.